packages/synapse-react-client/src/components/MoreResources/MoreResources.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Stack, Typography, useTheme, Link } from '@mui/material'
+import { Link as RouterLink } from 'react-router'
+
+type ResourceConfig = {
+  title: string
+  description: string
+  link: string
+}
+
+export type MoreResourcesProps = {
+  resources: ResourceConfig[]
+}
+
+const MoreResources = ({ resources }: MoreResourcesProps) => {
```

Comment:
If we extract the title, is this still a good name for this component? These are just basic cards with descriptions and links, they don't have to represent whatever 'resource' means to the ALS portal